print/print-apps/geoportailv3/a4_portrait.jrxml

```diff
@@ -12,8 +12,6 @@
 	<parameter name="mapSubReport" class="java.lang.String"/>
 	<parameter name="scalebarSubReport" class="java.lang.String"/>
 	<parameter name="name" class="java.lang.String"/>
-	<parameter name="legend" class="net.sf.jasperreports.engine.data.JRTableModelDataSource"/>
-	<parameter name="legendSubReport" class="java.lang.String"/>
```

Comment:
Why is a4 portrait the only file you've changed? Don't the other files deserve the same treatment?